vm.c

```diff
@@ -994,6 +994,14 @@ struct collect_outer_variable_name_data {
     bool isolate;
 };
 
+#if SIZEOF_VOIDP > SIZEOF_LONG
+# define ID2NUM(id) ULL2NUM(id)
```

Comment:
Even thought it's internal, I think we should follow `RB_` prefix for new macros.